examples/resources/auth0_organization_members/resource.tf

```diff
@@ -0,0 +1,21 @@
+resource "auth0_user" "user_1" {
+  connection_name = "Username-Password-Authentication"
+  email           = "{{.testName}}1@auth0.com"
```

Comment:
Can we replace the usage of `{{.testName}}` in this example with a sample value instead? I think it'll render better docs.